src/App.js

```diff
@@ -0,0 +1,72 @@
+import {
+	Routes,
+	Route,
+} from "react-router-dom";
+import "./App.css";
+import {
+	Header,
+	Footer,
+	Cart,
+	HomePage,
+	ProductListing,
+	Login,
+	Signup,
+	ForgotPassword,
+	UserPage,
+	WishList,
+} from "./components/index";
```

Comment:
you can just import from `./components`, that `/index` can be skipped.

Reply:
Okay, I'll do that.